lib/rules/no-shadow-route-definition.js

```diff
@@ -20,6 +20,10 @@ function buildRouteErrorInfo(routeInfo) {
   return `"${routeInfo.name}" (${routeInfo.fullPath}, ${buildSourceLocationString(routeInfo)})`;
 }
 
+function isNotNestedRouteWithSamePath(routeInfo) {
```

Comment:
Can we remove the negative from the title? I think this function would be more generic if we remove the negative and then call it like `!isNestedRouteWithSamePath()`

Reply:
Done.